macOS/DuckDuckGo/Fire/Model/Fire.swift

```diff
@@ -606,30 +605,17 @@ final class Fire: FireProtocol {
 
         case .window(tabCollectionViewModel: let tabCollectionViewModel, selectedDomains: _, _):
             visits = tabCollectionViewModel.localHistory
-            // clear tabs navigation history
```

Comment:
moved to ` tabCollectionViewModel.clearLocalHistory` including clearing the "removed tabs history"